src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +55,46 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/* Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
```

Comment:
```suggestion
/**
 * Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
```

This way it's marked as jsdoc.